Batchbald_implementation.ipynb

```diff
@@ -0,0 +1,395 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    plt.rcParams["axes.titlesize"] = 10
</pre><p>Why size 10? we are keeping max size 9 because it matches with caption.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Size 10 seemed to match for the figures generated. I'll change it to 9